File: lib-content-scrapers/src/main/java/com/ustadmobile/lib/contentscrapers/ck12/practice/AnswerResponse.java
package com.ustadmobile.lib.contentscrapers.ck12.practice;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class AnswerResponse {

    @SerializedName("instance")
    public Instance instance;

    public class Instance{

        public String solution;

        boolean multiAnswers;

        String questionTypeName;

        public List<AnswerObjects> responseObjects;

        public List<Object> answer;

        public class AnswerObjects {

            String isCorrect;

            public int displayOrder;

            public String displayText;

            public String orderText;

            public String optionKey;

            public int ansSeq;

        }

    }

}

File: lib-content-scrapers/src/main/kotlin/com/ustadmobile/lib/contentscrapers/ck12/practice/AnswerResponse.kt
package com.ustadmobile.lib.contentscrapers.ck12.practice

import com.google.gson.annotations.SerializedName

class AnswerResponse {

    @SerializedName("instance")
    var instance: Instance? = null

    inner class Instance {

        var solution: String? = null

        internal var multiAnswers: Boolean = false

        internal var questionTypeName: String? = null

        var responseObjects: List<AnswerObjects>? = null

        var answer: MutableList<Any>? = null

        inner class AnswerObjects {

            internal var isCorrect: String? = null

            var displayOrder: Int = 0

            var displayText: String? = null

            var orderText: String? = null

            var optionKey: String? = null

            var ansSeq: Int = 0

        }

    }

}
